app/code/Magento/CustomerGraphQl/etc/graphql/di.xml

```diff
@@ -20,4 +20,60 @@
             </argument>
         </arguments>
     </type>
+    <preference for="Magento\CustomerGraphQl\Api\DataMapperInterface" type="Magento\CustomerGraphQl\Model\Customer\DataMapper\DataMapperPool"/>
```

Comment:
Some logic here is redundant. Please have a look at `Magento/BundleGraphQl/etc/graphql/di.xml:27` as example.